iac-extensions/arm/src/main/java/org/sonar/iac/arm/tree/api/bicep/IfCondition.java

```diff
@@ -22,7 +22,7 @@
 import org.sonar.iac.arm.tree.api.Expression;
 import org.sonar.iac.arm.tree.api.ObjectExpression;
 
-public interface IfCondition extends Expression {
+public interface IfCondition extends Expression, HasKeyword {
```

Comment:
Regarding my previous comments: Expression already extends ArmTree. HasKeyword is not directly related to ArmTree including methods like `textRange` and so on.